src/request.rs

```diff
@@ -149,6 +149,16 @@ impl<State> Request<State> {
         self.req.host()
     }
 
+    /// Get the request MIME.
+    ///
+    /// This gets the request `Content-Type` header.
+    ///
+    /// [Read more on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)
+    #[must_use]
+    pub fn get_mime(&self) -> Option<Mime> {
```

Comment:
I don't believe this has a counterpart in http-types?

Reply:
@yoshuawuyts Right - but `tide::Response::set_mime()` exists, and `http_types` has the `content_type` methods on each.

Personally, I don't like "mime", and think "content type" is clearer, but a lot of people know it as mime, some specs and rfcs, etc...

Reply:
We had a long-ish conversation about this on `http-types` and chose to go with `content_type`. I don't really love it either, but think it's the right choice since "mime" is ambiguous (do we set the encoding, or do we set the accept?)